sky/provision/kubernetes/utils.py

```diff
@@ -58,7 +58,10 @@
 
 # Port-forward proxy command constants
 PORT_FORWARD_PROXY_CMD_TEMPLATE = 'kubernetes-port-forward-proxy-command.sh'
-PORT_FORWARD_PROXY_CMD_PATH = '~/.sky/kubernetes-port-forward-proxy-command.sh'
+# We add a version suffix to the port-forward proxy command to ensure backward
+# compatibility and avoid overwriting the older version.
+PORT_FORWARD_PROXY_CMD_PATH = ('~/.sky/'
+                               'kubernetes-port-forward-proxy-command-v2.sh')
```

Comment:
nit: we can save the version of the proxy command in constants, if we expect it can change in the future

Reply:
Ahh yeah we don't have a constants for kubernetes yet, but I've moved it to it's own variable for now